draco/synthesis/mockcatalog.py

```diff
@@ -997,6 +997,60 @@ def qso_velocity_error(z, rng):
 
         return dv
 
+    @staticmethod
+    def qsoalt_velocity_error(z, rng):
+        """Draw random velocity errors for quasars using a redshift dependent model.
+
+        This is based on the Lyke et al. model use in `qso_velocity_error` but fixing an
+        issue with the fraction of quasars in the wide distribution at all redshifts, and
+        reducing the errors at low redshift to account for the behaviour seen in Figure
+        9 on Lyke et al.
+
+        Parameters
+        ----------
+        z : np.ndarray
+            True redshift for the object.
+        rng : numpy.random.Generator
+            Numpy RNG to use for generating random numbers.
+
+        Returns
+        -------
+        dv: np.ndarray[nsample,]
+            Velocity errors in km / s.
+        """
+        QSO_SIG1_highz = 150.0
+        QSO_SIG1_lowz = 90.0
+        QSO_SIG2 = 1000.0
+
+        QSO_F_highz = 35.0
+        QSO_ztrans = 1.0
+        QSO_zwidth = 0.05
+
+        def smooth_step_function(z, zt, zw, fl, fh):
+            return (1 + np.tanh((z - zt) / zw)) * (fh - fl) / 2 + fl
+
+        def invfz(z):
+            return smooth_step_function(z, QSO_ztrans, QSO_zwidth, 0, 1 / QSO_F_highz)
```

Comment:
If I'm reading this right, it looks like the prefactor of the (1+tanh(...)) factor ends up being 1/70, whereas it is 1/35 in Eq. (131) of the stacking paper. I assume the code is correct and there is a typo in the paper?

Reply:
I'm saying that what is written in the code differs from Eq. (131) of the stacking paper by a factor of 2 in the prefactor

Reply:
I see. Let me check. Clearly the sims are done with what's in here. I'll need to check that the fit redshift error model matches that though....

Yes, we're good. `smooth_step_function` has the same definition in my comparison plots so it's just a typo in that equation.

Reply:
Maybe the one above it too. Will check.